docs/docs/apis/popup.md

```diff
@@ -19,7 +19,9 @@ Popups are identified by a caller-specified ID that should be unique.
 
 ## Types
 ``` javascript
-type PopupPosition  = 'top' | 'right' | 'bottom' | 'left';
+// 'context' mode makes it attempt to behave like a context menu -- defaulting
+// to the lower right of the anchor element and working its way around.
+type PopupPosition  = 'top' | 'right' | 'bottom' | 'left' | 'context';
```

Comment:
Can 'context' be used in a list with the other positions, or must it always be by itself as the only requested position?